packages/examples/cvat/exchange-oracle/src/core/types.py

```diff
@@ -72,6 +75,11 @@ class RecordingOracleEventTypes(str, Enum, metaclass=BetterEnumMeta):
     task_rejected = "task_rejected"
 
 
+class ReputationOracleEventTypes(str, Enum, metaclass=BetterEnumMeta):
+    # TODO: rename to ReputationOracleEventType
```

Comment:
What is this comment? Do you want to rename this specific enum?

Reply:
All of them, really. It's unconventional for enums to have plural name, however to be consistent with current code base I went with it for now.

As for renaming it in this PR, I didn't want to introduce unnecessary noise which would be inevitable, if I was to rename all of the enums.

Reply:
Which naming convention do you mean? This project currently uses plurals for enum names.

Reply:
I wasn't referencing any specific convention within our project. Rather, I was referring to the general standard in Python (and other languages for that matter), where classes, including enums, are typically named using singular nouns in `PascalCase`. 

You can see example of following this convention here: https://github.com/python/cpython/issues/104659.

I understand that our codebase currently uses plurals for enum names (and seen the change in history from once singular variant), so I've maintained that consistency for now. However, considering the broader community standards, we might want to think about transitioning back to singular names. This would make our code more intuitive for new contributors who are familiar conventions. Hence the `# TODO`.

As I said, I chose not to make this change in the current PR to avoid introducing unnecessary  changes.